### ISSUE TITLE
Feature request: allow to upload a contact picture in the create user dialog

### ISSUE THREAD

**Issue by j-ed** [nextcloud/server#2867]
## Expected behavior
If an administrator creates a new user account, he should be able to upload a contact picture.

## Current behavior
If an administrator creates a new user account, he cannot upload a contact picture.

### Steps to reproduce
1. Login as as an administrator
2. Choose Users -> Create user

#### Server Configuration
OS: Linux 3.2.82
Web server: Apache2 2.4.25
Database: MariaDB 5.5.53
PHP version: 5.6.29
Nextcloud version: 10.0.2

#### Client Configuration
Browser: Mozilla Firefox 50.1.0
Operating system: Windows 7

**Comment by Bugsbane** [comment]
Agreed. I set up quite a few accounts for family members, and it's definitely cumbersome how for each I have to log out, log in to their account, uplkoad an avatar from there (and enter their email) and then log back into my account afterwards.

Not sure if this merits another issue filed, but I would add that the entering their user email is the other half of the user information that can't be created when making a new user.

**Comment by MariusBluem** [comment]
What do you guys think @nextcloud/designers 🤔

**Comment by skjnldsv** [comment]
Agreed!

**Comment by Bugsbane** [comment]
> What do you guys think @nextcloud/designers 🤔

Well, I clearly agreed already... 😉

**Comment by MariusBluem** [comment]
> Well, I clearly agreed already... 😉

I just wanted to hear some more opionions on this topic ... I didn't misread your comment if this is your comment about 😁

**Comment by Bugsbane** [comment]
Oh, absolutely. I don't presume to speak for the entire designers team. Just wasn't sure if you were aware I was on it.

**Comment by skjnldsv** [comment]
@j-ed could you help on this? :)
:+1:

**Comment by szaimen** [comment]
Related to https://github.com/nextcloud/server/pull/23529

**Comment by joshtrichards** [comment]
Related: #49979
